### ISSUE TITLE
Add/remove department by Head / Admin / Super Admin

### ISSUE THREAD

**Issue by rapee** [youpin-city/youpin-api#247]
# TL;DR

There will be no "Assigned" and "Processing" on frontend anymore. 
The difference is only if pin has `assigned_department` or not.

1. `assigned_department` must be changeable regardless of its **status**.

2. I post as `organization_admin` but cannot remove department from pin. Both of the following request don't change anything. :(

```
PATCH /pins/<id>
Body: { $set: { assigned_department: null } }
```

```
PATCH /pins/<id>
Body: { $unset: { assigned_department: '' } }
```

Look at this: [Permission Table](https://docs.google.com/document/d/1ZQvvsPC4j0BXLQgnZxsFHviW5q-VQE0oEYez5DE9mRA/edit)

**Comment by supasate** [comment]
Based on discussion, we will allow all roles except normal user to change state between pending and assigned.

**Comment by parnurzeal** [comment]
From discussion, it seems this is more on allowing anyoen to change assigned_department field.
It should be fixed by PR #278
